ReactSkia/components/RSkComponent.h

```diff
@@ -77,6 +78,7 @@ class RSkComponent : public RnsShell::Layer, public std::enable_shared_from_this
   void requiresLayer(const ShadowView &shadowView);
   void updateProps(const ShadowView &newShadowView , bool forceUpdate);
   virtual void updateComponentProps(const ShadowView &newShadowView,bool forceUpadate) = 0;
+  virtual bool onHandleKey(rnsKey  eventKeyType){return false;};
```

Comment:
Just to make it more clear the return value can  tell if key is handled or not and extra out parameter "stopPropagating" to be filled by component.